api/types.proto

```diff
@@ -315,6 +315,17 @@ message UpdateConfig {
 	// roll back to the previous service spec. If the MaxFailureRatio
 	// threshold is hit during the rollback, the rollback will pause.
 	float max_failure_ratio = 5;
+
+	// RolloutMode controls the order of operations when rolling out an
+	// updated task. Either the old task is shut down before the new task
+	// is started, or the new task is started before the old task is shut
+	// down.
+	enum RolloutMode {
```

Comment:
Worried this is confusing: stop first / start first.

This has an effect of the order of operations concerning the replacement (start new first vs stop old first), rather than just start vs stop